hack/lldb/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Some overlap here with https://github.com/containers/bootc/pull/593 a bit and of course - a lot of overlap with https://github.com/containers/podman-bootc ...I was really thinking about trying to make that a more generic tool, but it gets tricky.

There's *so many* virtualization wrappers on Linux...

Reply:
agreed, eventually podman-bootc should provide everything needed for this, but in the meantime I just wanted to use lldb. The primary (only?) thing preventing using podman-bootc is the networking stuff.